api/api.go

```diff
@@ -30,6 +30,10 @@ const (
 	// the HTTP token.
 	HTTPTokenEnvName = "CONSUL_HTTP_TOKEN"
 
+	// HTTPTokenFileEnvName defines an environment variable name which sets
+	// the HTTP token file.
+	HTTPTokenFileEnvName = "CONSUL_HTTP_TOKEN_FILE"
```

Comment:
Not saying we need to change anything but there are some other aspects of our CLI that use an '@' prefix to denote that something should be loaded from a file instead of using the string raw. We could do something similar here and reduce the possible env vars we have to care about. Just something to think about.